src/App.tsx

```diff
@@ -12,155 +13,30 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={({ isActive }) =>
+              cn('navbar-item', { 'has-background-grey-lighter': isActive })
+            }
```

Comment:
Would be good to create separate function for this.

Reply:
Good point, fix it